### ISSUE TITLE
Install `nightly` toolchain as `nightly-$date` also

### ISSUE THREAD

**Issue by huonw** [brson/multirust#9]
Or provide an option to do it.

This would assist with e.g. bisecting bugs, because upgrades would leave a trail of successively nightlies.

**Comment by brson** [comment]
Good idea. To do this cleanly we need the bots to upload more metadata to associate the current nightly with an archive date. We _could_ crawl the dates in the vicinity of the current date for checksums that match but that's pretty gnarly.

**Comment by brson** [comment]
Probably want to symlink when on platforms where that's possible.

**Comment by brson** [comment]
In fact, under this scheme we would probably only install to the dated directory and always make nightly/beta/stable a symlink (except maybe on windows).

**Comment by brson** [comment]
I might instead make 'stable', 'nightly', and 'beta' special cases that use their archive toolchain directory to avoid having to deal with symlinks on windows.

**Comment by brson** [comment]
Before this con be fixed in rust-buildbot where the manifests are created, I would be open to just having multirust probe recent dates for matching checksums to discover the archive date.
